### DISCUSSION TITLE
Exposure Time range

### DISCUSSION THREAD

**Discussion by Dizlz** [raspberrypi/picamera2#695] [Q&A]
I've spent a lot of time but still can't find a way to set the value range for the exposure time.  I need to set the exposure time range in python. The "Exposure Time" function sets a strict value. I would like to set a range of values. For example minExp = 1000 maxExp = 5000

**Comment by davidplowman** [comment]
You can set a range of exposure values by setting up exposure modes in the camera tuning file (see section 5.8 in the [tuning manual](https://datasheets.raspberrypi.com/camera/raspberry-pi-camera-guide.pdf)), and then you could select that (or another one) dynamically while the camera is running. But I don't think there's an easy way to set a pair of arbitrary limits while the camera is running.
